TransactionManager/back_end/client_app/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.status import (
+    HTTP_204_NO_CONTENT,
+    HTTP_400_BAD_REQUEST,
+)
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.authentication import TokenAuthentication
 
-# Create your views here.
+from .models import Client
+from .serializers import AClientSerializer
+
+class User_permissions(APIView):
```

Comment:
I believe you've already made this `User_permissions` class. Instead of creating it in every app it may be a good idea to import it. Don't forget your DRY principle

Reply:
Thank you! I didn't know I could do that.  Will do that for all apps.  
What do you think about this: create a `permissions.py` file in user_app with just:
```
from rest_framework.views import APIView
class User_permissions(APIView):
    authentication_classes = [TokenAuthentication]
    permission_classes = [IsAuthenticated]
```
then, in every views file that requires it:
`from user_app.permissions import User_permissions`

Reply:
Wilco.

Reply:
Done.